packages/experimental/Avatar/src/Avatar.helpers.ts

```diff
@@ -0,0 +1,126 @@
+  physicalSize: number;
+  iconSize: number;
+  iconStrokeWidth: number;
+  initialsSize: number;
+};
+
+const sizeTable: { [P in AvatarSize]: AvatarSizeConfig } = {
+  size8: { physicalSize: 8, iconSize: 0, iconStrokeWidth: 0, initialsSize: 4 },
+  size24: { physicalSize: 24, iconSize: 8, iconStrokeWidth: 2, initialsSize: 10 },
+  size32: { physicalSize: 32, iconSize: 8, iconStrokeWidth: 2, initialsSize: 12 },
+  size40: { physicalSize: 40, iconSize: 12, iconStrokeWidth: 2, initialsSize: 14 },
+  size48: { physicalSize: 48, iconSize: 12, iconStrokeWidth: 2, initialsSize: 16 },
+  size56: { physicalSize: 56, iconSize: 16, iconStrokeWidth: 3, initialsSize: 18 },
+  size72: { physicalSize: 72, iconSize: 20, iconStrokeWidth: 3, initialsSize: 20 },
+  size100: { physicalSize: 100, iconSize: 28, iconStrokeWidth: 4, initialsSize: 36 },
+  size120: { physicalSize: 120, iconSize: 32, iconStrokeWidth: 4, initialsSize: 40 },
+};
+
+export function getSizeConfig(size: AvatarSize): AvatarSizeConfig {
+  return sizeTable[size];
+}
+
+const colorTableFluent: { [P in AvatarColor]: string } = {
+  cornflower: globalTokens.color.cornflower.primary,
+  blue: globalTokens.color.blue.primary,
+  royalBlue: globalTokens.color.royalBlue.primary,
+  teal: globalTokens.color.teal.primary,
+  forest: globalTokens.color.forest.primary,
+  darkGreen: globalTokens.color.darkGreen.primary,
+  berry: globalTokens.color.berry.primary,
+  hotPink: globalTokens.color.hotPink.primary,
+  grape: globalTokens.color.grape.primary,
+  purple: globalTokens.color.purple.primary,
+  pumpkin: globalTokens.color.pumpkin.primary,
+  red: globalTokens.color.red.primary,
+  burgundy: globalTokens.color.burgundy.primary,
+  orchid: globalTokens.color.orchid.primary,
+  brass: globalTokens.color.brass.primary,
+  darkRed: globalTokens.color.darkRed.primary,
+  beige: globalTokens.color.beige.primary,
+  platinum: globalTokens.color.platinum.primary,
+  steel: globalTokens.color.steel.primary,
+  brown: globalTokens.color.brown.primary,
+};
+
+/**
+ * Converts the AvatarColor into a hex color value
+ */
+export function convertCoinColorFluent(coinColor: AvatarColor): string {
+  return colorTableFluent[coinColor];
+}
+
+export function calculateEffectiveSizes(tokens: AvatarTokens): AvatarSizeConfig {
+  const { size, avatarSize, iconSize, iconStrokeWidth, initialsSize } = tokens;
+
+  if (size) {
+    return sizeTable[size];
+  } else {
+    const {
+      physicalSize: defaultPhysicalSize,
+      iconSize: defaultIconSize,
+      iconStrokeWidth: defaultIconStrokeWidth,
+      initialsSize: defaultInitialsSize,
+    } = sizeTable['size40'];
+
+    return {
+      physicalSize: avatarSize || defaultPhysicalSize,
+      iconSize: iconSize || defaultIconSize,
+      iconStrokeWidth: iconStrokeWidth || defaultIconStrokeWidth,
+      initialsSize: initialsSize || defaultInitialsSize,
+    };
+  }
+}
+
+export function getRingThickness(thickness: RingThickness): number {
```

Comment:
Hm.. I wonder if these options are still the same for win32. This would be a place where we should get input from design

Reply:
I'm not sure, that's why I didn't change anything in PersonaCoin. I found the design spec but we don't have it for RN. I asked Design